cli_reference/basic_cli_operations.adoc

```diff
@@ -754,3 +717,22 @@ link:https://access.redhat.com/errata/RHSA-2015:1650[For security purposes], the
 the command is executed by a `cluster-admin` user. Administrators can SSH into
 a node host, then use the `docker exec` command on the desired container.
 ====
+
+[[troubleshooting-cli]]
+
+== Troubleshooting oc
+At any point in time you can get more verbosed output from any command by increasing
```

Comment:
s/At any point in time you can get more verbosed output/You can get more verbosed output